### PR TITLE
Merge PostMVP into FutureFeatures for clarity.

### PR DESCRIPTION
**Pull request by flagxor**

### CODE REVIEW

**Comment by s3ththompson** [comment]
I like this idea. Do we want to consider replacing "MVP" into "initial version" by the way? Although that word really helped us focus on the initial set of features, I feel as though it's somewhat confusing to explain in the context of the browser preview and/or launch. what do others think?

**Comment by jfbastien** [comment]
I wouldn't rename "MVP" because that's what's already all over the internet.

This change looks good to me.

File: README.md [hunk 1]
```diff
@@ -32,9 +32,8 @@ The actual WebAssembly specification is being developed in the [spec repository]
 
 We've mapped out features we expect to ship:
 
- 1. In [the Minimum Viable Product (MVP)](MVP.md);
- 2. Closely [after the MVP](PostMVP.md);
- 3. In [future versions](FutureFeatures.md).
+ 1. In initial [Minimum Viable Product (MVP)](MVP.md) release;
```
Comment (s3ththompson) [inline README.md]: nit: "In an initial..." or "In the initial..."

**Comment by s3ththompson** [comment]
@jfbastien good point. changing things for clarity always runs the risk of the change itself being more confusing than the original :)

**Comment by lukewagner** [comment]
lgtm

**Comment by sunfishcode** [comment]
FutureFeatures.md (as it currently stands) is a collection of ideas, some strong, others vague, speculative, or contentious, meant to be things to think about. PostMVP.md is meant to address a specific audience of people who perceive the MVP feature set as incomplete even for its intended initial use cases. Merging the two, even with stars for priorities, may blur the PostMVP message, and may lend undue weight to some of the items in FutureFeatures today.

**Comment by flagxor** [comment]
Dan, would marking some of these with :radioactive: address your concern about lending too much weight to some?

**Comment by flagxor** [comment]
Or perhaps :confused: (confused) ?

**Comment by flagxor** [comment]
Or :clown: (clown) ...

**Comment by flagxor** [comment]
Or :chestnut: (chestnut) ... or :rocket: (rocket).... Anyhow.

**Comment by sunfishcode** [comment]
My comment is that PostMVP addresses a difference audience than FutureFeatures (in its present state), and that keeping it as a separate page feels like it addresses each in a way that a single list, even with annotations, doesn't. However, if others disagree, I won't object further.

**Comment by flagxor** [comment]
Adding the unicorn tags lead me to notice that the two are mixed together a good deal (and confusing).
I think if we retain the separation we'll probably want to move a few things from FutureFeatures into PostMVP and probably have two separate icons for links leading to the two.

What do others think?

**Comment by jfbastien** [comment]
I'd rather merge the two.

**Comment by s3ththompson** [comment]
@sunfishcode I agree that the bucketing between "near-term, we're definitely adding these" and "aspirational far-reaching future". however, I don't think that "post-mvp" as a term particularly captures the fact that they'll be implemented imminently.

How about changing the introduction to introduce two explicit buckets: "On deck for immediate design" and "Future proposals". Everything with stars would be in the first bucket. Everything afterwards would be in the second?

File: FutureFeatures.md [hunk 2]
```diff
@@ -1,17 +1,99 @@
...
 These are features that make sense in the context of the
-[high-level goals](HighLevelGoals.md) of WebAssembly but are not considered part
-of the [Minimum Viable Product](MVP.md) or the essential
-[post-MVP :unicorn:](PostMVP.md)
-feature set which are expected to be standardized immediately after the
-MVP. These will be prioritized based on developer feedback, and will be
+[high-level goals](HighLevelGoals.md) of WebAssembly but won't be part of the
+initial [Minimum Viable Product](MVP.md) release.
+
+We except a several essential features to be standardized immediately after the
```
Comment (s3ththompson) [inline FutureFeatures.md]: expect

Comment (flagxor) [inline FutureFeatures.md]: done

**Comment by flagxor** [comment]
Revised to separate into two sections.
Keeping the stars.
Dan would shuffling some of the more controversial stuff downward help?

**Comment by rossberg-chromium** [comment]
LGTM

I for one think that replacing "MVP" with "initial version" would be much better. Outside the small circle of people that have followed this repo very closely I have yet to encounter a single person that knows what the heck I'm talking about when I use a term like "MVP". It's rather off-putting management/sales bla, TBH. I think we can happily risk the minor confusion among the few early people for the benefit of making it clearer to the broader future audience.

**Comment by jfbastien** [comment]
"MVP" was a core thing in the initial messaging. I think we should keep it. It's late to change it now.

**Comment by titzer** [comment]
lgtm

**Comment by jfbastien** [comment]
Assuming this is good to merge, given the timing and all.

**Pull request merged**